pygeo/parameterization/DVGeo.py

```diff
@@ -2481,7 +2481,7 @@ def writeRefAxes(self, fileName):
            extension will be added.
         """
         # Name here doesnt matter, just take the first one
-        self.update(self.points.keys()[0], childDelta=False)
+        self.update(list(self.points.keys())[0], childDelta=False)
```

Comment:
The order is not guaranteed in Python <3.7, so to be safe I would suggest sorting the list before accessing `[0]`. That way, we are sure all procs have the same name.

Also not sure if this is a concern here or not, I'm not familiar with this part of the code.

Reply:
I don't think it's concern here because we usually call the write functions on a single proc anyway (the underlying `writeTecplot1D` function from pySpline is serial). There are other places in DVGeo where we cast the keys to a list where it might be more important.

In any case, we only support and test Python >= 3.7 right?

Reply:
Yeah fair, if this is expected to be run on a single proc then we are okay. Like you said, this issue might exist elsewhere but that's a separate problem.

Yes, but at the same time I hesitate to rely on behaviour which is built into the underlying CPython implementation rather than something which is built into the language, if that makes sense. In the future they could change this behaviour again and I'd rather be safe than sorry.